### ISSUE TITLE
PyPi main page not available

### ISSUE THREAD

**Issue by adiroiban** [mmabey/vbox-sdk#1]
Thanks for maintaining this package!

https://pypi.python.org/pypi/vbox-sdk gives a "Not found" error

but I can access a specific version https://pypi.python.org/pypi/vbox-sdk/5.1.18

Can you please look at that is going on there.

Thanks!

**Comment by mmabey** [comment]
Hi @adiroiban! Wow, I didn't think anyone was using this project. To answer your question directly, the reason why you get the "Not found" error is that I configured PyPI to hide the releases I uploaded. There are two reasons why I did this.

1. I believe this was the first package I ever uploaded to PyPI, and I wasn't very graceful at figuring out the learning curve. This resulted in there not being any downloads for version `5.1.18` because I didn't get everything right the first time. But because of how PyPI works, you can't re-upload the same version of a package twice, so the downloads are available for that version under [`5.1.18-0`](https://pypi.python.org/pypi/vbox-sdk/5.1.18-0) (like I said, not very graceful). So it was kind of embarrassing to have that made public.

2. The bigger reason is that I realized that when you install VirtualBox, it also installs the same files I included in this repo. It seemed like a lot of extra work for me to monitor the [release page](http://download.virtualbox.org/virtualbox/), download and extract the SDK, move the files into the local clone of the repo, make and commit a new release (tag) on GitHub, and upload everything to PyPI, when people could get the same library just by simply installing VirtualBox. I know there are ways to automate this build process, but I'm presently finishing up my dissertation and don't have the time to set it all up.

I suppose that there are some use cases where you don't want to install VirtualBox on the machine where you plan to run your Python code that interfaces with VirtualBox, but since that fell outside the scope of my own use cases when I created this project, I figured it was simpler to just effectively close shop.

If you would care to help with the project, I would be happy to accept it. If you were to do all of the steps listed in item 2 above except for uploading to PyPI, I could easily do the rest. Otherwise, I'll go ahead and close the issue as a WONTFIX.

**Comment by adiroiban** [comment]
I am using the Python SDK from a remote machine, via the SOAP webservice.

I ended up creating a package which contains  

```
vboxapi/__init__.py
vboxapi/VirtualBox_client.py
vboxapi/VirtualBox_constants.py
vboxapi/VirtualBox_services.py
vboxapi/VirtualBox_types.py
vboxapi/VirtualBox_wrappers.py
```

I think that is easier to just close this repo.

I will try to automate the release in another repo and once I am done I will come back :)

Thanks for you time!
